src/main/java/com/yacht/molvenorestaurant/business/Guest.java

```diff
@@ -1,15 +1,28 @@
 package com.yacht.molvenorestaurant.business;
 
-import lombok.Data;
+import java.util.ArrayList;
+import java.util.List;
 
-@Data
 public class Guest {
     private int roomnumber;
-
-    /**
-     * First and last name
-     */
     private String name;
+    return roomnumber;
+    }
+
+    public void setRoomnumber(int roomnumber) {
+        this.roomnumber = roomnumber;
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    public void setName(String name) {
+        this.name = name;
+    }
 
-    private boolean isFemale;
+    public void Guest(int roomnumber,String name){
```

Comment:
constructors horen meestal boven alle andere methods. Maar in dit geval gebruiken we geen contructors voor onze pojo's